docs/userguide/ut_run-script.md

```diff
@@ -1,52 +1,71 @@
-                 If not defined, then output will be displayed on screen, even if the parameter -s is not specified.
-                 If more than one -o parameter is specified for one -f parameter, the last one is taken into consideration.
-  -s           - Forces putting output to to screen for a given -f parameter.
-  -c           - If specified, enables printing of test results in colors as defined by ANSICONSOLE standards
+  user              - username to connect as
+  password          - password of the user
+  database          - database to connect to
+  -p=ut_path(s)     - A path or a comma separated list of paths to unit test to be executed.     
+                      The path can be in one of the following formats:
+                        schema[.package[.procedure]]
+                        schema:suite[.suite[.suite][...]][.procedure]
+                      Both formats can be mixed in the comma separated list.
+                      If only schema is provided, then all suites owner by that schema (user) are executed.
+                      If -p is omitted, the current schema is used.
+  -f=format         - A reporter to be used for reporting.
+                    Available options:
+                      -f=ut_documentation_reporter
+                        A textual pretty-print of unit test results (usually   use for console output)
+                      -f=ut_teamcity_reporter
+                        A teamcity Unit Test reporter, that can be used to    visualize progress of test execution as the job    progresses.
+                      -f=ut_xunit_reporter
+                        A XUnit xml format (as defined at:    http://stackoverflow.com/a/9691131 and at    https://gist.github.com/kuzuha/232902acab1344d6b578)
+                        Usually used  by Continuous Integration servers like   Jenkins/Hudson or Teamcity to display test results.
+                      -f=ut_coverage_html_reporter
+                        Generates a HTML coverage report providing summary    and detailed information on code coverage.
+                        The html reporter is based on open-source    simplecov-html reporter for Ruby.
+                        It includes source code of the code that was covered   (if possible).
+                      -f=ut_coveralls_reporter
+                        Generates a JSON coverage report providing detailed    information on code coverage with line numbers.
+                        This coverage report is designed to be consumed by    cloud services like https://coveralls.io/.
+                      -f=ut_coverage_sonar_reporter
+                        Generates a JSON coverage report providing detailed    information on code coverage with line numbers.
+                        This coverage report is designed to be consumed by    local services like https://about.sonarqube.com/.
```

Comment:
I would replace the line
This coverage report is designed to be consumed by    local services like https://about.sonarqube.com/.
with something like:
This report is designed to be consumed by [SonarQube](https://about.sonarqube.com/) coverage.